dateparser/search/search.py

```diff
@@ -180,7 +174,7 @@ def __init__(self):
         self.search = ExactLanguageSearch(self.loader)
 
     def detect_language(self, text, languages):
-        if isinstance(languages, (list, tuple, Set)):
+        if isinstance(languages, (list, tuple, set)):
```

Comment:
I think this is technically a backward-incompatible change, as it is technically possible for an object to be an instance of a class inheriting from `Set` that is not a `set`.